src/all.css

```diff
@@ -0,0 +1,13 @@
+.all-box {
```

Comment:
object modifier names should be clearer in context. For the sake of making the point clear: which box is this selector affecting? **All** boxes? or _only_ some boxes of the web document?